docs/05.policy/08.responserules/08.responserules.md

```diff
@@ -13,7 +13,7 @@ Response Rules provide a flexible, customizable rule engine to automate response
 
 Creating a new Response Rule using the following:
 
-+ Group. A rule will apply to a Group. Please see the section Run-Time Security Policy -> Groups for more details on Groups and how to create a new one if needed.
++ Group. A rule will apply to a container Group (address group is not supported, for example criteria address=x.x.x.x/x). Please see the section Run-Time Security Policy -> Groups for more details on Groups and how to create a new one if needed.
```

Comment:
```suggestion
+ Group. A rule will apply to a container Group (address group is not supported, for example, criteria address=x.x.x.x/x). Please see the section [Run-Time Security Policy -> Groups](../04.groups/04.groups.md) for more details on Groups and how to create a new one if needed.
```

Reply:
👍

Reply:
Hi @gtam , are you able to make the update to the content? I can merge in once updated, thank you!